### PR TITLE
Fix enabling/disabling compliance flag for Voltron

### PR DESCRIPTION
**Pull request by rene-dekker**
Based on most situations the flag to was incorrectly set to enable compliance forwarding in Voltron.

### CODE REVIEW

**Review by Brian-McM — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Review by doublek — APPROVED** [review]
LGTM

**Pull request merged**
